workers/ohsome_quality_analyst/api.py

```diff
@@ -56,26 +30,74 @@ async def load_bpolys(bpolys: str) -> None:
 class IndicatorParameters(BaseModel):
     bpolys: Optional[str] = None
     dataset: Optional[str] = None
-    featureId: Optional[Union[int, str]] = None
+    featureId: Union[int, str, None] = None
     layerName: Optional[str] = None
     fidField: Optional[str] = None
 
 
 class ReportParameters(BaseModel):
     bpolys: Optional[str] = None
     dataset: Optional[str] = None
-    featureId: Optional[Union[int, str]] = None
+    featureId: Union[int, str, None] = None
     fidField: Optional[str] = None
 
 
+def empty_api_response() -> dict:
+    return {
+        "attribution": {
+            "url": "https://ohsome.org/copyrights",
+            "text": "© OpenStreetMap contributors",
+        },
+        "apiVersion": oqt_version,
+    }
+
+
+def int_or_str_param_type(value: str) -> Union[int, str]:
+    """Parse parameter as Interger otherwise parse as String"""
+    try:
+        return int(value)
+    except (ValueError, TypeError):
+        if isinstance(value, str):
+            return value
+        else:
+            raise ValueError("Given parameter is not a valid integer or string")
```

Comment:
Do you have an example where this could be required? As far as I see, this is only used for the `feature_id`. The feature id given by a GET/POST request should always be given as a string, afaik. If you want to prevent the parameter from being optional, I would check/set this with fastAPI.

Reply:
Yes you are right. The else statement is never reached. Except maybe when None is passed as argument. But before calling this function it is checked whether feature id is none or not.

Reply:
I meant in general, why do you want a variable which is either `int` or `str`? In the database you can only handle one of them, right? So you can handle all identifier in OQT as string, or did I miss a problem with that? I think it would be much easier to focus on **one** type for identifier :wink:

Reply:
Ah, now I see what you mean. 
Since https://github.com/GIScience/ohsome-quality-analyst/pull/37 `feature_id` can be str or int depending on what `feature_id_field` is.

E.g. 
```bash
# int
oqt create-indicator --fid-field ogc_fid --feature-id 1
# str
oqt create-indicator --fid-field name --feature-id heidelberg
```

What do you think about this behavior?